view/common/resource-values.phtml

```diff
@@ -23,7 +23,7 @@ $filterLocaleCallback = function ($value) use ($lang) {
 
 <dl>
 <?php
-foreach ($values as $term => $propertyData):
```

Comment:
Non blocking, but the linting cleanup may actually make it more difficult to compare our changes with the upstream version of this file over time. I'm not asking to change it back, but just noting my concern that we may want to keep track of whether forcing our coding standards on files that are sourced from others ends up making it more difficult to bring in changes from upstream in the future. (i.e. a diff with our file and upstream if they make a change we want to consider bringing in will be more difficult to reason with now that we've applied non-functional changes to our forked version)

Reply:
Yeah, I worry about this too. The origin of this is, from my perspective, that Omeka isn't following their own code standards. [Their documentation](https://omeka.org/s/docs/developer/contributing/#code-style) calls for people to follow the PSR-2 style guide, which is what we've configured our tools to enforce. However, as in this case, the templates aren't always correctly indented - and there's a persistent quirk where they omit spaces around `if () : {` constructs.

It may be that their internal tooling is somehow configured differently than ours (they use php-cs-fixer, where we use phpcodesniffer). It is something to look at systemically down the road, because ultimately I agree with you that we should be able to run a diff between our theme templates and the core files and see what we've changed.

Reply:
I should add that I'm hesitant to disable the linting for things like code indenting, because that's been one of the most easily introduced sources of cognitive friction with developers in the past. Given the choice, I'd rather have to worry about indenting changes when looking backwards to the core files if it means that we can avoid discussions about proper indenting levels for all changes into the future.

If I can find the rule for spaces between `()` and `:` characters, I'm willing to disable that check, though.

Reply:
It's one of the weirdest rules I've seen to date. `if () : {` is very odd compared to `if (): {`

This was entirely non-blocking and at this point if we've been correcting all the other forked files, not doing it on this one makes no sense. I wasn't aware that more files were forked than not.